include/rocksdb/utilities/backupable_db.h

```diff
@@ -142,6 +142,14 @@ struct BackupableDBOptions {
   }
 };
 
+struct CreateBackupOptions {
+  // If false, background_thread_cpu_priority is ignored.
+  bool enable_update_background_thread_cpu_priority = false;
+  // Currently only effective on Linux, value ranging from -20 to 19,
+  // 19 is the lowest priority.
+  int background_thread_cpu_priority = 0;
```

Comment:
Hmm.. This is too platform dependent and I'm not sure it is a good idea. Can we avoid it for now until we need it? We can always do it for lowest level for now. If one day we need it, we may consider what the option should be.

Reply:
According to the person who requests this feature, for each backup, he wants to be able to update both sched type (such as SCHED_IDLE) and priority. I think this requirement is specific for Linux.

Reply:
created a new enum `CpuPriority` to be platform independent